spring-context/src/main/java/org/springframework/context/support/PropertySourcesPlaceholderConfigurer.java

```diff
@@ -24,13 +24,7 @@
 import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
 import org.springframework.beans.factory.config.PlaceholderConfigurerSupport;
 import org.springframework.context.EnvironmentAware;
-import org.springframework.core.env.ConfigurablePropertyResolver;
-import org.springframework.core.env.Environment;
-import org.springframework.core.env.MutablePropertySources;
-import org.springframework.core.env.PropertiesPropertySource;
-import org.springframework.core.env.PropertySource;
-import org.springframework.core.env.PropertySources;
-import org.springframework.core.env.PropertySourcesPropertyResolver;
+import org.springframework.core.env.*;
```

Comment:
As [Spring Coding Style](https://github.com/spring-projects/spring-framework/wiki/Code-Style#import-statements), Wildcard import is forbidden.

Reply:
updated, thanks @cuspymd.